### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by MichalMaler** [comment]
Please, Robert, can you help me with fixing the navbar and ID entry? Can't spot the error myself. :(
Need to rework the intro, please, don't merge this baby yet.

File: modules/installation-guide/nav.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@
 ** xref:configuring-storage-types.adoc[]
 ** xref:importing-untrusted-tls-certificates.adoc[]
 ** xref:switching-between-external-and-internal-communication.adoc[]
-** xref:setting-up-keycloak-che-username-readonly-theme.adoc[]
+** xref:setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc[]
```
Comment (themr0c) [inline modules/installation-guide/nav.adoc:46]: Please rename the  page in the pages directory to the same name.

File: antora-playbook.yml [hunk 2]
```diff
@@ -91,6 +91,7 @@ asciidoc:
     prod-host: che-host
     prod-id-short: che
     prod-id: eclipse-che
+    prod2-id: eclipse-che
```
Comment (themr0c) [inline antora-playbook.yml:94]: Could we avoid this new unfriendly attribute?

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+// configuring-che
+
+//ID versions:
+
+// setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page
+
+// setting-up-the-{identity-provider-id}-{prod-id-short}-username-readonly-theme-for-the-{prod2-id}-login-page-for-the-{prod2-id}-login-page
+
+[id="setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page_{context}"]
+= Setting up the {identity-provider} {prod-id-short}-username-readonly theme for the {prod2} login page
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:12]: Let's avoid `prod2`, too. What is the reason for having such attributes?

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The following procedure is relevant for all {prod-short} instances with the enabled OpenShift OAuth service.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:14]:
```suggestion
The following procedure is relevant for all {prod-short} instances with the OpenShift OAuth service enabled.
```

File: antora-playbook.yml (hunk 2 above)
Comment (MichalMaler) [reply antora-playbook.yml:94]: what makes it unfriendly?

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+After a user log in to {identity-provider} for the first time using the OpenShift credentials, a page allowing the user to update the account information displays. The user can change his username that was automatically created during the first login, but then a change to the Keycloak settings need to be done. Keeping the changed username without the steps mention below will prevent the user's workspaces from running.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:16]:
```suggestion
When a user logs in to {identity-provider} for the first time using OpenShift credentials, a page allowing the user to update account information is displayed. It is possible to change the username that was automatically created during the first login, but then {identity-provider} settings must be modified. Failing to modify {identity-provider} settings prevents the user's workspaces from running.
```

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+
+.Prerequisites
+
+* An instance of {prod-short} running in OpenShift.
+* A user is logged in to the {identity-provider} service.
+
+.Procedure
+
+After changing a username, set the Login Theme option to `readonly`.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:26]:
```suggestion
After changing a username, set the *Login Theme* option to `readonly`.
```

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+. In the main *Configure* menu on the left, select *Realm Settings*
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:28]:
```suggestion
. In the main *Configure* menu on the left, select *Realm Settings*:
```

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
++
+image::keycloak/{project-context}-keycloak-username-readonly-theme.png[link="../_images/keycloak/{project-context}-keycloak-username-readonly-theme.png"]
+
+. Navigate to the *Themes* tab.
+
+. In the *Login Theme* field, select the `{prod-id-short}-username-readonly` option and click btn:[Save] to apply the changes.
```
Comment (rkratky) [inline modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:34]:
```suggestion
. In the *Login Theme* field, select the `{prod-id-short}-username-readonly` option and click the btn:[Save] button to apply the changes.
```

**Review by rkratky — CHANGES_REQUESTED** [review]

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc (hunk 3 above)
Comment (MichalMaler) [reply modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:12]: In some cases, I needed to have Codeready Workspaces and Eclipse Che.

I can make this proc work without it.

File: antora-playbook.yml (hunk 2 above)
Comment (rkratky) [reply antora-playbook.yml:94]: > what makes it unfriendly?

The `2` is unclear.

File: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc (hunk 3 above)
Comment (rkratky) [reply modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc:12]: We have `prod-short` -- what's wrong with using "Che" and "CodeReady Workspaces"?

File: antora-playbook.yml (hunk 2 above)
Comment (rkratky) [reply antora-playbook.yml:94]: > The reason for prod2 was - I need Codeready Workspace and Eclipse Che / NOT / Red Hat Codeready Workspaces and Eclipse Che.

Right, but why not use "CodeReady Workspaces" and "Che"?

>  Maybe one day we should even reconsider using Eclipse Che as prod-short value, to have the prod-short map work between up and down. These inconsistencies in the product ui are just so painful.

No, that would be inconsistent. "Eclipse", like "Red Hat", is the brand, and "Che", like "CodeReady Workspaces", is the app name. We either use both, or none -- but not the complete name in upstream and incomplete in downstream.

**Review by tolusha — APPROVED** [review]

**Pull request merged**